internal/hud/hud.go

```diff
@@ -166,6 +166,12 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 				}
 			case r == 'R': // hidden key for recovering from printf junk during demos
 				h.r.screen.Sync()
+			case r == ' ':
```

Comment:
yes this is silly but if you leave a comment `// [space] - trigger build for selected resource` this code gets 50% more readable

Reply:
done